src/main/java/com/safecharge/request/GetPaymentPageRequest.java

```diff
@@ -0,0 +1,157 @@
+package com.safecharge.request;
+
+import com.safecharge.model.Item;
+import com.safecharge.model.UrlDetails;
+import com.safecharge.request.builder.SafechargePPBuilder;
+import com.safecharge.util.Constants;
+import com.safecharge.util.ValidChecksum;
+import com.safecharge.util.ValidationUtils;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.util.ArrayList;
+import java.util.List;
+
+@ValidChecksum(orderMappingName = Constants.ChecksumOrderMapping.API_GENERIC_CHECKSUM_MAPPING)
+public class GetPaymentPageRequest extends SafechargeRequest {
```

Comment:
Hi! Thanks for the effort. Did you follow the API guide from safecharge.com? 

I am not sure this pull request can be merged as you don't have all the background details. This class should extend SafechargeOrderDetailsRequest. Also name it GetPaymentPageUrlRequest.